<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsViewUpdateTimes</class>
 <widget class="QFrame" name="CSettingsViewUpdateTimes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>304</width>
    <height>77</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <layout class="QVBoxLayout" name="vl_UpdateTimes">
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gb_ViewUpdateTimes">
     <property name="title">
      <string>View update times</string>
     </property>
     <layout class="QGridLayout" name="gl_ViewUpdateTimes">
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <property name="spacing">
       <number>4</number>
      </property>
      <item row="0" column="3">
       <widget class="QSlider" name="hs_AtcUpdateTime">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="tb_ResetUser">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../misc/misc.qrc">
          <normaloff>:/pastel/icons/pastel/16/arrow-refresh-small.png</normaloff>:/pastel/icons/pastel/16/arrow-refresh-small.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="lbl_UserUpdateTime">
        <property name="toolTip">
         <string>User update time (5-30s)</string>
        </property>
        <property name="text">
         <string>User</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QSlider" name="hs_UserUpdateTime">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbl_AircraftUpdateTime">
        <property name="toolTip">
         <string>Aircraft update time (5-30s)</string>
        </property>
        <property name="text">
         <string>Aircraft</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSlider" name="hs_AircraftUpdateTime">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QSlider" name="hs_RenderingUpdateTime">
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbl_RenderingUpdateTime">
        <property name="toolTip">
         <string>Rendering update time (5-30s)</string>
        </property>
        <property name="text">
         <string>Rendering</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbl_AtcUpdateTime">
        <property name="toolTip">
         <string>ATC update time (5-30s)</string>
        </property>
        <property name="text">
         <string>ATC</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>lbl_AircraftUpdateTime</zorder>
     <zorder>hs_AircraftUpdateTime</zorder>
     <zorder>lbl_RenderingUpdateTime</zorder>
     <zorder>hs_RenderingUpdateTime</zorder>
     <zorder>lbl_AtcUpdateTime</zorder>
     <zorder>hs_AtcUpdateTime</zorder>
     <zorder>lbl_UserUpdateTime</zorder>
     <zorder>hs_UserUpdateTime</zorder>
     <zorder>tb_ResetUser</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../misc/misc.qrc"/>
 </resources>
 <connections/>
</ui>
